src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/MoveStaticMembersDialog.xaml

```diff
@@ -0,0 +1,105 @@
+﻿<vs:DialogWindow
```

Comment:
We should run the dialog through our accessibility check tool. Reach out to me or @ryzngard this week to talk you through it.